release/tasks/calculate-new-versions.ts

```diff
@@ -0,0 +1,76 @@
+import { Context, Bump, Tag, Package } from '../@types/custom';
+import * as semver from 'semver';
+
+import { debug } from '../lib/utils';
+import updateDependencies from '../lib/update-dependencies';
+
+/**
+ *
+ * @param tag The tag of the commit
+ */
+const commitType = (tag: Tag): Bump => {
+
+    switch (tag) {
+        case 'Docs': return Bump.none;
+        case 'Build': return Bump.none;
+        case 'Upgrade': return Bump.patch;
+        case 'Chore': return Bump.patch;
+        case 'Fix': return Bump.patch;
+        case 'New': return Bump.minor;
+        case 'Breaking': return Bump.major;
+        default: return Bump.none;
+    }
+};
+
+
+const calculatePackageNewVersion = (pkg: Package, bump: Bump): string => {
+
+    if (!pkg.publishedVersion) {
+        return pkg.content.newVersion;
```

Comment:
Should this be covered in the types somewhere? `pkg.content` is currently `any` (which means `pkg.content.newVersion` is `any` as well).

Reply:
This is most likely a typo. I think it should be `pkg.content.version`